cli/src/pcluster/api/controllers/cluster_operations_controller.py

```diff
@@ -447,3 +451,13 @@ def _get_yaml_path(path, parameter):
     if parameter:
         yaml_path.append(parameter)
     return ".".join(yaml_path)
+
+
+def _get_plugin_metadata(cluster):
+    """Get the metadata name and version used for the response of DescribeCluster."""
+    if cluster.config.scheduling.scheduler == "plugin":
+        full_metadata = get_attr(cluster.config, "scheduling.settings.scheduler_definition.metadata")
```

Comment:
if for any reason full_metadata is None let's return None rather than failing on `.get`

Reply:
Change to
```
    def _get_plugin_metadata(self):
        """Get the metadata name and version used for the response of DescribeCluster when the scheduler is plugin."""
        full_metadata = get_attr(self.config, "scheduling.settings.scheduler_definition.metadata")
        return Metadata(name=full_metadata.get("Name"), version=full_metadata.get("Version")) if full_metadata else None
```